modules/connatixBidAdapter.js

```diff
@@ -41,10 +42,28 @@ export function getBidFloor(bid) {
   }
 }
 
+export function validateBanner(mediaTypes) {
+  if (!mediaTypes[BANNER]) {
+    return true;
+  }
+
+  const banner = deepAccess(mediaTypes, BANNER, {});
+  return (Boolean(banner.sizes) && isArray(mediaTypes[BANNER].sizes) && mediaTypes[BANNER].sizes.length > 0);
+}
+
+export function validateVideo(mediaTypes) {
+  if (!mediaTypes[VIDEO] || !mediaTypes[VIDEO].context) {
+    return true;
+  }
+
+  const video = deepAccess(mediaTypes, VIDEO, {});
+  return video.context !== 'adpod';
```

Comment:
context I think it should be required in media types.

please consult other adapters and see how they handle it

Reply:
Some adapters require context to be defined, some don't, while the documentation says it's only recommended. The context will be defaulted to `instream` from our back-end, as far as I know, so no need to make it required.